data/migrations/v1.3.0-line.sql

```diff
@@ -0,0 +1,34 @@
+UPDATE planet_osm_line
+  SET mz_road_level = mz_calculate_min_zoom_roads(planet_osm_line.*)
+  WHERE
+      (tags -> 'highway' IN ( 'trunk_link', 'primary_link', 'secondary_link', 'tertiary_link',
+                   'residential', 'unclassified', 'road', 'living_street', 'pedestrian',
+                   'path', 'track', 'cycleway', 'bridleway', 'footway', 'steps',
+                   'service' ) OR
+       tags -> 'whitewater' = 'portage_way')
+      AND mz_calculate_min_zoom_roads(planet_osm_line.*) IS NOT NULL;
+
+CREATE INDEX new_planet_osm_line_roads_geom_index ON planet_osm_line USING gist(way) WHERE mz_road_level IS NOT NULL;
```

Comment:
It shouldn't be necessary to re-create the indexes, they should be updated along with the rows in the `UPDATE` statement above.